src/native/profiler.cc

```diff
@@ -647,6 +676,16 @@ void Profiler::prepare_scope(std::string& scope) {
     std::replace(scope.begin(), scope.end(), '.', '/');
 }
 
+void Profiler::add_search_scope(string &scope)
```

Comment:
Can you add `static`?

Reply:
But it's already `static`: https://github.com/Decave/JCoz/blob/92f9257fe5eac044d7406ed2803db2aa01f4180f/src/native/profiler.h#L173
By the way, I see that the parameters list instead requires some modification, so I'll add it.